timesketch/lib/analyzers/domain.py

```diff
@@ -69,16 +70,28 @@ def run(self):
         satellite_emoji = emojis.get_emoji('SATELLITE')
         for domain, count in domain_counter.iteritems():
             emojis_to_add = [satellite_emoji]
+            tags_to_add = []
             text = '{0:s} seen {1:d} times'.format(domain, count)
 
+            is_known_cdn = utils.get_cdn_providers(domain)
+            if is_known_cdn:
+                tags_to_add.append('known-cdn')
+                for cdn in is_known_cdn:
+                    cdn_counter[cdn] += 1
+
             for event in domains.get(domain, []):
+                event.add_tags(tags_to_add)
                 event.add_emojis(emojis_to_add)
                 event.add_human_readable(text, self.NAME, append=False)
-                event.add_attributes({'domain_count': count})
+                new_attributes = {'domain_count': count}
+                if is_known_cdn:
+                    new_attributes['cdn_provider'] = ' '.join(is_known_cdn)
```

Comment:
why not keep it just as a list?

then you can do checks to see if CDN is in that list...

Reply:
See above. Refactored this in the utility function instead.